docs/octopus-rest-api/octo.exe-command-line/index.md

```diff
@@ -29,8 +29,8 @@ We provide a number of ways to install Octo onto your machine:
 - **[delete-releases](/docs/octopus-rest-api/octo.exe-command-line/delete-releases.md)**:  Deletes a range of releases.
 - **[deploy-release](/docs/octopus-rest-api/octo.exe-command-line/deploy-release.md)**:  Deploys a release.
 - **[dump-deployments](/docs/octopus-rest-api/octo.exe-command-line/dump-deployments.md)**:  Writes deployments to an XML file that can be imported in Excel.
-- **[export](/docs/octopus-rest-api/octo.exe-command-line/export.md)**:  Exports an object to a JSON file.
-- **[import](/docs/octopus-rest-api/octo.exe-command-line/import.md)**:  Imports an Octopus object from an export file.
+- **[export](/docs/octopus-rest-api/octo.exe-command-line/export.md)**:  Exports an object to a JSON file. Deprecated. Please see https://g.octopushq.com/DataMigration for alternative options.
```

Comment:
Can @Octobob format these links? i.e., [Data Migration](https://g.octopushq.com/DataMigration).

Reply:
I cant have nice names on the links, but we can make the whole url a link.
Fixed by https://github.com/OctopusDeploy/docs/pull/493